atc/api/jobserver/badge_test.go

```diff
@@ -0,0 +1,251 @@
+package jobserver_test
```

Comment:
Existing tests for the badge endpoint are here: https://github.com/concourse/concourse/blob/97393cea938d4dfb125071b9542b9e151eab4a47/atc/api/jobs_test.go#L533

I think most of your tests here are already covered by the existing tests. Can you move your tests alongside the existing tests for the badge endpoint?